crates/alerter/src/lib.rs

```diff
@@ -221,14 +295,10 @@ impl Alerter {
             .await
             .context("solvable_orders")?
             .into_iter()
-            .filter(|order| {
-                !order.is_liquidity_order()
-                    && !order.partially_fillable
-                    && !matches!(order.status, OrderStatus::PresignaturePending)
```

Comment:
Safe to skip removing `OrderStatus::PresignaturePending` since those are already filtered by autopilot [here](https://github.com/cowprotocol/services/blob/main/crates/autopilot/src/solvable_orders.rs#L158-L159)